.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,14 @@
+<!-- 
+STOP -- PLEASE READ
+
+GitHub is not the right place for support requests.
+
+If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
+
+You can also post your question on the [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubenetes](https://discuss.kubernetes.io/) forum.
+
```

Comment:
nit: typo - r missing in "Kubernetes"

Reply:
cc: @nikhita

Reply:
IMO this looks good now and includes all support channels. :+1: